test/ragnarok/ManagedPortfolio.test.ts

```diff
@@ -72,11 +77,45 @@ describe('ManagedPortfolio', () => {
       const creationTimestamp = (await provider.getBlock(deployTx.blockHash)).timestamp
       expect(await portfolio.endDate()).to.equal(creationTimestamp + YEAR)
     })
+
+    it('sets owner', async () => {
```

Comment:
This test is duplicated

Reply:
looks like somebody moved it to top and it merged